migrations/versions/0480_move_heartbeat_templates.py

```diff
@@ -0,0 +1,136 @@
+                        VALUES ('{NOTIFY_HEARTBEAT_SERVICE_ID}', 'GCNotify Heartbeat', '{NOW}', True, False, 20000, 20000, 5000000, 5000000, False, False, False, 'central', 'gc.notify.heartbeat.notification.gc',
+                        '{NOTIFY_USER_ID}', 1)
+                     """
+    op.execute(heartbeat_service_history_insert)
+
+    for send_type in ('sms', 'email'):
+        heartbeat_service_permissions_insert = f"""INSERT INTO service_permissions (service_id, permission, created_at) VALUES ('{NOTIFY_HEARTBEAT_SERVICE_ID}', '{send_type}', '{NOW}')"""
+        op.execute(heartbeat_service_permissions_insert)
+
+    # Copy the service permissions from the existing Notify service to the new Heartbeat service.
+    perms_insert = f"""
+        INSERT INTO permissions (id, service_id, user_id, permission, created_at)
+            SELECT uuid_in(md5(random()::text)::cstring), '{NOTIFY_HEARTBEAT_SERVICE_ID}', user_id, permission, '{NOW}'
+              FROM permissions
+             WHERE service_id = '{NOTIFY_SERVICE_ID}'
+    """
+    op.execute(perms_insert)
+
+    # The annual billing is required for new services. Let's copy the annual billing
+    # data from existing Notify service to the new heartbeat service.
+    annual_billing_insert = f"""
+        INSERT INTO annual_billing 
+        (id, service_id, financial_year_start, free_sms_fragment_limit, created_at, updated_at) 
+         SELECT uuid_in(md5(random()::text)::cstring), '{NOTIFY_HEARTBEAT_SERVICE_ID}', financial_year_start, free_sms_fragment_limit, created_at, updated_at 
+           FROM annual_billing
+          WHERE service_id = '{NOTIFY_SERVICE_ID}'
+          ORDER BY financial_year_start DESC
+          LIMIT 1
+    """
+    op.execute(annual_billing_insert)
+
+    user_to_service_insert = f"""
+        INSERT INTO user_to_service 
+            (user_id, service_id)
+            SELECT user_id, '{NOTIFY_HEARTBEAT_SERVICE_ID}'
+              FROM user_to_service
+             WHERE service_id = '{NOTIFY_SERVICE_ID}'
+    """
+    op.execute(user_to_service_insert)
+
+    for template_id in TEMPLATE_IDS:
+        op.execute(
+            TEMPLATE_UPDATE.format(
+                NOTIFY_HEARTBEAT_SERVICE_ID,
+                template_id
+            )
+        )
+
+        op.execute(
+            TEMPLATE_HISTORY_UPDATE.format(
+                NOTIFY_HEARTBEAT_SERVICE_ID,
+                template_id
+            )
+        )
+
+
+def downgrade():
+    annual_billing_delete = f"""DELETE FROM annual_billing WHERE service_id = '{NOTIFY_HEARTBEAT_SERVICE_ID}'"""
+    op.execute(annual_billing_delete)
+
+    user_to_service_delete = f"""DELETE FROM user_to_service WHERE service_id = '{NOTIFY_HEARTBEAT_SERVICE_ID}'"""
```

Comment:
This line was updated as well to delete all records in the table for the service heartbeat service.